.github/workflows/ci.yml

```diff
@@ -28,6 +29,9 @@ jobs:
         with:
           node-version: lts/*
 
+      - name: Set up node ignore-scripts config
+        run: npm config set ignore-scripts false
```

Comment:
I was trying to debug the sentry-cli issue that occurs in the in the build process in github action but it did not work and this will be removed. It was one of the things mentioned here https://github.com/getsentry/sentry-webpack-plugin/issues/120